src/graph.py

```diff
@@ -338,23 +346,23 @@ def output_forest_as_networkx(self, vertex_indices, edge_indices):
         Construct a networkx graph from a set of vertex and edge indices (i.e. a pcsf output)
 
         Arguments:
-            vertex_indices (list): indices of the vertices selected in self.nodes
+            vertex_indices (list): indices of the vertices selected in self.nodes. Note, this list must be of type int or boolean. Errors may arise from empty numpy arrays of dtype='object'
```

Comment:
An annoying error can be produced when lists are passed in with type object. Normally this is does not happen, since the robustness and specificity dataframes cast `node_indices` as ints. However, these dataframes are empty, or the merge produces and empty df, the values are automatically set to type object. 

The result is that the function appears to break when `vertex_indices` is empty. I added it since it took me a long time to debug, and I could see this being an issue in the future.